hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/ha/MockSCMHAManager.java

```diff
@@ -107,6 +108,14 @@ public void shutdown() throws IOException {
     ratisServer.stop();
   }
 
+  @Override
+  public List<String> getRatisStatus() {
+    return Arrays.asList(
```

Comment:
You can use the default SCM RatisServer port. Check ratisBindPort in SCMHAConfiguration

Reply:
Switched to use the default port. In the future I will think about how to better construct `MockSCMHAManager` for testing purpose (as the code is evolving we can keep updating `MockSCMHAManager`)